docs/upgrade.html

```diff
@@ -47,14 +47,77 @@
 
 <p><b>Note:</b> Bumping the protocol version and restarting can be done any time after the brokers were upgraded. It does not have to be immediately after.
 
-<ol>
+<h5><a id="upgrade_1020_streams" href="#upgrade_1020_streams">Updating a Kafka Streams Application</a></h5>
+<ul>
     <li>Upgrading a Kafka Streams Applications:
         <ul>
-            <li>You need to recompile your code. Just swapping the jar file will not work and will break your appliation.</li>
-            <li>If you use a custom timestamp extractor, you will need to update this code, because the <code>TimestampExtractor</code> interface was changed.</li>
+            <li> You need to recompile your code. Just swapping the jar file will not work and will break your application. </li>
+            <li> <code>KStreamBuilder#table()</code> always requires to specify a store name. </li>
+            <li> <code>KTable#through()</code> always requires to specify a store name. </li>
+            <li> If you use a custom timestamp extractor, you will need to update this code, because the <code>TimestampExtractor</code> interface was changed. </li>
+            <li> If you register custom metrics, you will need to update this code, because the <code>StreamsMetric</code> interface was changed. </li>
+            <li> Upgrading your Streams application from 0.10.1 to 0.10.2 does not require to upgrade your brokers.
```

Comment:
> does not require to upgrade your brokers.

Should be 
`does not require you to upgrade your brokers.`

There are other similar grammatical issues.